app/src/main/java/ingsw/group1/findmyphone/managing/Manager.java

```diff
@@ -115,23 +117,28 @@ public void analyzeRequest(String requestMessage, String phoneNumber) {
      */
     public void activeResponse(SMSMessage messageResponse, Class activityClass) {
         String requestMessage = messageResponse.getData();
-        if (locationManager.isLocationRequest(requestMessage)
-                || alarmManager.isAlarmRequest(requestMessage)) {
-            openRequestsActivity(requestMessage, messageResponse.getPeer().getAddress(), activityClass);
-        }
-
-        //The only expected response
-        if (locationManager.isLocationResponse(requestMessage)) {
-            Double longitude;
-            Double latitude;
-            try {
-                longitude = Double.parseDouble(locationManager.getLongitudeFrom(requestMessage));
-                latitude = Double.parseDouble(locationManager.getLatitudeFrom(requestMessage));
-                locationManager.openMapsUrl(currentContext, latitude, longitude);
-            } catch (Exception e) {
-                //Written in log for future users to report
-                Log.e(MANAGER_TAG, e.getMessage());
-            }
+        switch (MessageBuilder.getMessageType(messageResponse)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5068369)